### ISSUE TITLE
keyword_case: false positive for FINAL METHODS

### ISSUE THREAD

**Issue by fdupret** [abaplint/abaplint#3832]
<img width="586" height="220" alt="Image" src="https://github.com/user-attachments/assets/1c1874ca-a057-4b56-8ed8-4c1109dc150b" />

```abap
REPORT zfoo.

INTERFACE lif.
  METHODS meth1.
  METHODS meth2.
ENDINTERFACE.

CLASS lcl_foo DEFINITION ABSTRACT.
  PUBLIC SECTION.
    INTERFACES lif
      ABSTRACT METHODS
      meth1
      FINAL METHODS
      meth2.
ENDCLASS.
```
`Identifiers should be lower case: "FINAL"(keyword_case)`

ABSTRACT METHODS is not reported, as expected. I expected the same for FINAL METHODS.
